src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -413,6 +413,7 @@ public override void M<U>(U u)
             comp3.VerifyDiagnostics();
         }
 
+
         [Fact]
         public void InterfaceAttributes()
```

Comment:
>ERR_NoSuchMemberOrExtension [](http://example.com/codeflow?start=37&length=27)

Just curious: Why are we reporting `ERR_NoSuchMemberOrExtension` in addition to `ERR_MissingDeconstruct` now?

Reply:
@jcouv for thoughts, but my guess is that we suppressed semantic diagnostics in the presence of syntactic diagnostics.  Now these are all semantic and we tell you about these problems, even if you're in the wrong language version.